week-4/InClass/B-objects-get-set/exercise-4.js

```diff
@@ -2,14 +2,14 @@
     Write code in the space provided so that the expected values output
 */
 
-let dog = {
-    name: 'Billy',
-    wantsToPlay: false
-};
 
-// WRITE CODE BELOW THIS LINE
 
+// WRITE CODE BELOW THIS LINE
 
+let dog = {
+    name: 'Rex',
+    wantsToPlay: true
+};

```

Comment:
How could you reassign new values to the object properties without re-writing the whole object? i.e. with keeping the original object as it was.